src/components/DatasetOverviewTab/data/definitions.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "pdc": "Modified",
+      "definition": "Most recent date on which the dataset was changed, updated or modified"
+    },
+    {
+      "pdc": "Released",
+      "definition": "xxx"
+    },
+    {
+      "pdc": "Issued",
+      "definition": "Dare of formal issuance\" Project Open Data uses lable \"Release"
```

Comment:
These need some proofreading. I know these are coming from the ticket directly but this is not coherent. We need someone to modify the expected definitions.

Reply:
I agree. Here is an attempt of me editing the text but the author of the file should probably the person editing the definitions.

https://github.com/GetDKAN/cmsds-open-data-components/pull/239/commits/8fc5e776fdead285469a17e3bf62b26add10f62e

Reply:
since the text is abstracted into a json file, we will create a new ticket to edit the text when the final version of the text is ready.